modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase

```diff
@@ -19,6 +19,37 @@ definition {
 		PortalInstances.tearDownCP();
 	}
 
+	@description = "LPS-159838. Can enable walkthrough in System Settings."
+	@priority = "5"
+	test CanBeEnabled {
+		property portal.acceptance = "true";
+
+		var portalURL = PropsUtil.get("portal.url");
+
+		task ("Given Walkthrough in System Settings") {
+			JSONGroup.addGroup(groupName = "Site Name");
+
+			ApplicationsMenu.gotoSite(site = "Site Name");
+		}
+
+		task ("When walkthrough is enabled") {
+			Walkthrough.enableWalkthrough(fileName = "walkthrough_configuration.json");
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1",
+				widgetName = "JS Walkthrough Sample");
```

Comment:
Same for here, I just want to check that you can check mark "Enable Walkthrough" without inputting JSON steps and save.

Reply:
Although now that I think about it, does it make sense to have the option to enable walkthrough if no steps are provided? :thinking:

Reply:
[Fixed here ](https://github.com/liferay-frontend/liferay-portal/pull/2721)